packages/react-core/src/components/Dropdown/DropdownToggleCheckbox.tsx

```diff
@@ -25,6 +28,8 @@ export interface DropdownToggleCheckboxProps
   id: string;
   /** Aria-label of the checkbox */
   'aria-label': string;
+  /** Aria-label for the default progress icon */
+  defaultProgressArialabel?: string;
```

Comment:
Nit: for consistency, it could be worth matching the lettercase of similar arialabel props:

```suggestion
  defaultProgressAriaLabel?: string;
```